trappy/plotter/AbstractDataPlotter.py

```diff
@@ -73,16 +74,22 @@ def _parse_value(self, signal_def):
         :type signal_def: str
         """
 
-        event, column = signal_def.split(":")
+        match = re.match("(?P<event>[^:]+):(?P<column>[^:]+)(:.+)?", signal_def)
```

Comment:
Maybe better to use group names for consistency with other fields?

This should enforce also a check on color format:
```python
match = re.match("r(?P<event>[^:]+):(?P<column>[^:]+)(?P<color>:[0-9]+,[0-9]+,[0-9]+)?", signal_def)
#....
color = match.group("color")
if color:
    color = [int(n) for n in color[1:].split(",")]
else
    color = None
```